package-dev/Runtime/SentryInitialization.cs

```diff
@@ -31,7 +31,11 @@ public static void Init()
 #if SENTRY_NATIVE_IOS
                 SentryNativeIos.Configure(options);
 #elif SENTRY_NATIVE_ANDROID
+#if ENABLE_IL2CPP
+                SentryNativeAndroid.Configure(options, true);
+#else
                 SentryNativeAndroid.Configure(options);
+#endif
```

Comment:
What about we remove the default argument and always pass a value?

```suggestion
                var il2cpp = false;
#if ENABLE_IL2CPP
               il2cpp = true;
#endif
                SentryNativeAndroid.Configure(options, il2cpp);
```

Reply:
What about
```csharp
var il2cpp = 
#if ENABLE_IL2CPP
        true;
#else
        false;
#endif
``` 
?